packages/transaction-controller/src/types.ts

```diff
@@ -727,3 +727,10 @@ export type DefaultGasEstimates = {
    */
   maxPriorityFeePerGas?: string;
 };
+
+export type SecurityAlertResponse = {
```

Comment:
Love that we now have a type for this! ❤️ 

Is it worth adding some brief JSDoc?

Also can we use this type for the `securityAlertResponse` property in the `TransactionMetadata` type so we have a better understanding of the data?